app/utils/timeUtils.js

```diff
@@ -225,6 +225,26 @@ function getTimeDiff(startTime, endTime, unit, isFloat = false) {
   return moment(endTime).diff(moment(startTime), unit, isFloat);
 }
 
+/**
+ * getDateWithinTZ
+ * Get Date object with the default time-zone factored in
+ *
+ * @param {Moment | Date | string | undefined} value represents a specific date or undefined for the current moment
+ * @returns {Date} date object correspondent to the supplied value
+ */
+function getDateWithinTZ(value) {
```

Comment:
Why do we need this function? Why not `moment().toDate()`? https://momentjs.com/docs/#/displaying/as-javascript-date/

Reply:
Same question

Reply:
https://github.com/moment/moment-timezone/issues/309#issuecomment-193885034